### ISSUE TITLE
Application Request: simpleX

### ISSUE THREAD

**Issue by usernameisJim** [getumbrel/umbrel-apps#703]
We have a request for simpleX on Umbrel

https://github.com/simplex-chat/simplex-chat

**Comment by nmfretz** [comment]
Another request here: https://github.com/getumbrel/umbrel-apps/issues/764

**Comment by jeremycady** [comment]
+1

**Comment by ascisco** [comment]
yes please

**Comment by arguser** [comment]
This request would be for both the `smp-server` and the `xftp-server`?

**Comment by jeremycady** [comment]
I would say so. XFTP for the file transfers.

**Comment by arguser** [comment]
I'm on it :)

**Comment by arguser** [comment]
Here's the PR for SimpleX Server https://github.com/getumbrel/umbrel-apps/pull/1510 
Please take a look an comment any suggestion. I've made it to work through TOR which would require connection your client to TOR network.

**Comment by jjmmbb** [comment]
Have someone tried it? It's available on:

https://github.com/escapingnetwork/umbrel-community-app-store

**Comment by jeremycady** [comment]
> Have someone tried it? It's available on:
> 
> https://github.com/escapingnetwork/umbrel-community-app-store

That one has not worked for me.

**Comment by arguser** [comment]
@jeremycady yes, I've had it working via TOR on a previous version not sure which commit. Now I'm now trying an updated version I had to build (there's no official multiarch image) with some changes and couldn't find time to make it work again.

**Comment by arguser** [comment]
@jeremycady try again, give it a sec to restart then use the address and replace the @umbrel for @public-ip, remember to port forward. Didn't try with tunneling yet but shouldn't be too difficult.

**Comment by jjmmbb** [comment]
Is it working on TOR?

**Comment by arguser** [comment]
@jjmmbb no, it was working on TOR but been asked to go vanilla for a first release.

**Comment by jeremycady** [comment]
@arguser I was finally able to get everything working over the local network. 

Here are the issues I ran into.

First, I thought I initially used the port that the app was using for the "website". That was 9092. That was incorrect.

I found the actual parts in the docker-compose.yml, 5223 for smp and 5443 for the xftp. If those could be listed on the "homepage" that will help others connect much easier.

So I replaced umbrel with IPADDRESS:5223 and IPADDRESS:5443

On the first install on umbrel, I could connect xftp but not smp in the android app. Even using the correct ports. For the life of me, I couldn't figure out the smp. But, then, I caught a %3D in the address.

My assumption is that %3D was just part of the random address assigned, but, when putting it into the Simplex App on my phone, it was decoding as '=' by the app.

I reinstalled the app on umbrel to get assigned a new random address. The new address worked and I've connected multiple devices over the local network. 

I'll try cloudflare tunnel later, when I get an opportunity.

**Comment by jeremycady** [comment]
@arguser I cannot get cloudflare to work yet. I'm going to have to keep playing with it and there's not a lot of documentation on using smp or xftp via tunnels.

I tested Tailscale and it works great.

**Comment by arguser** [comment]
> [@arguser](https://github.com/arguser) I was finally able to get everything working over the local network.
> 
> I found the actual parts in the docker-compose.yml, 5223 for smp and 5443 for the xftp. If those could be listed on the "homepage" that will help others connect much easier.

Done!
